### ISSUE TITLE
rhash not reading the full path from own generated digest file for some files.

### ISSUE THREAD

**Issue by s-i-m-s** [rhash/RHash#133]
Why does it only see `transcoded.mp4` in this specific line rather than the full path

I'm running `rhash -cM '/home/user/Desktop/badline.txt'` I have attached the file in question in the case that the space is actually a special character of some sort.

**Comment by rhash** [comment]
Looks like a bug in a line parsing, should be investigated deeper. I've reproduced it with the file:
~~~
d41d8cd98f00b204e9800998ecf8427e FTP/nplayer/Downloads/one/senate transcoded.mp4
~~~

As a workaround, for now, you can use one of the following correctly handled formats:
~~~
FTP/nplayer/Downloads/one/senate transcoded.mp4 d41d8cd98f00b204e9800998ecf8427e
MD5( FTP/nplayer/Downloads/one/senate transcoded.mp4 ) = d41d8cd98f00b204e9800998ecf8427e
~~~

**Comment by rhash** [comment]
The string `FTP/nplayer/Downloads/one/senate` is detected by RHash as a 192-bit Tiger message digest in a Base64 format and "transcoded.mp4" as the actual filepath. So the line
~~~
d41d8cd98f00b204e9800998ecf8427e FTP/nplayer/Downloads/one/senate transcoded.mp4
~~~
is interpreted by RHash as `HEX(MD5(FILE)) BASE64(Tiger(FILE)) <FILEPATH>`.

Possible solution can be disabling of detection of base64 strings if a hex formatted string is already found in the same line.

**Comment by rhash** [comment]
Fixed in the master branch by commit 4478eca07dcb7817445565874997c670053b2e80.

**Comment by rhash** [comment]
The fix is included into RHash v1.4.1.
